### ISSUE TITLE
Download bootstrap during setup phase

### ISSUE THREAD

**Issue by Cryptoslave** [Anoncoin/oldrepo-backup#99]
I think we should have Anoncoin Qt have a question dialog on first use: "download bootstrap from anoncoin.net?" yes/no

Better would be if i2p router is there to download the torrent from I2P-snark

**Comment by mrkellysch** [comment]
I completely agree. Making things simple to do would increase usage. Also as a first time user I've been downloading blocks to my qt client (over 100 weeks out of sync)  and it has been over 3 days to finish. I know I could have downloaded the .dat files to speed up blocks installation but a warning of the fact that it would take this long to finish (eta) would have been advantageous, or option to download dat files. The only way to increase uptake is to make it numpty friendly for people like me. At the moment you have to be a technical genius to use it.

**Comment by Cryptoslave** [comment]
Which version did you build?

Normally it shall take about 12 hour to synch from scratch without bootstrap.dat and over I2P. Maybe your processor was not good enough? On VPS with one core only it takes more than 12 hours due to lots of calculus.

The issue can be one of four things:
-old version like 9.6.9 have a bug which make them waste both time and bandwidth to synch. This was corrected in 9.6.10.
-"debug=" is set in the anoncoin.conf, which enable the writing of a HUGE debug.log file.
-bad i2p connections if synching over i2p.
-a too old or not efficient enough processor (like a VPS 1 core, a celeron/atom or a raspeberry pi).

Thank you for your comment

**Comment by mrkellysch** [comment]
Last week I used the command $ git clone https://github.com/Anoncoin/anoncoin.git .I used the tutorial from here: [https://forum.anoncoin.net/viewtopic.php?f=1&t=479](url) . I presume cloning the anoncoin github repository downloads the latest stable version? I am running the OS from a USB 3.0 flash drive (cold storage). I don't know if this would be a problem as it's using the dual core i5 intel processor 2.50 GHz. If you care to look at the step by step tutorial linked to above there is no starting "debug="in terminal. I used clearnet as to reduce download time. Any ideas cryptoslave. Thanks for the response so far

**Comment by Cryptoslave** [comment]
Ah OK I was refering to the version 9.6.9 (https://wiki.anoncoin.net/How_to_build_Anoncoin_develop)

I do not know how much time bootstrap take with 8.5.6.

Maybe your USB key do not like the I/O operation due to read/write? 

Well I have no clue, try with 9.6.9 next time ;)
